lib/cdo/aws/dms.rb

```diff
@@ -65,5 +67,114 @@ def self.tasks(file)
         [task_name, {rules: rules}]
       end
     end
+
+    # Get list of Replication Tasks that should be run on a specified schedule.
+    # @param schedule [String] Frequency that task should be executed ('daily', 'weekly', etc.)
+    # Returns array of Aws::DatabaseMigrationService::Types::ReplicationTask
+    def self.replication_tasks(schedule)
```

Comment:
Judging by how many times `replication_task` is repeated in this collection of class methods and how many times `replication_task_arn` is the first argument, it seems like it could be simpler if rolled into a class (e.g., `Cdo::DMS::ReplicationTask`, with `arn` as the argument to `#initialize`) with a set of instance methods.

Reply:
Good point.  What started out as a couple of helper methods to find and start tasks is clearly better organized as a class.  Done!